lib/ansible/modules/monitoring/zabbix/zabbix_maintenance.py

```diff
@@ -288,6 +287,9 @@ def main():
             collect_data=dict(type='bool', required=False, default=True),
             timeout=dict(type='int', default=10),
         ),
+        required_one_of=[
+            ['host_names', 'host_groups']
```

Comment:
This is a breaking change. Right now, the module only requires one of them if `state == 'present'`. If `state == 'absent'`, both are not used at all.